src/tools/gophish_export.py

```diff
@@ -247,27 +266,38 @@ def main():
         )
     except ValueError:
         logging.critical(
-            '"{}"is not a valid logging level.  Possible values are debug, info, warning, and error.'.format(
-                log_level
-            )
+            f'"{log_level}"is not a valid logging level.  Possible values are debug, info, warning, and error.'
         )
         return 1
 
     else:
         # Connect to API
         try:
             api = connect_api(args["API_KEY"], args["SERVER"])
-            logging.debug("Connected to: {}".format(args["SERVER"]))
+            logging.debug('Connected to: {args["SERVER"]}')
```

Comment:
You are missing the f-string `f` here.

Reply:
This is showing as a f-string in fa6ca50.

Reply:
Fixed in commit 5222886.